package.json

```diff
@@ -15,7 +15,7 @@
     "start:debug": "nest start --debug --watch",
     "start:prod": "node dist/src/main",
     "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
-    "test": "run-s migration:test && NODE_ENV=test jest -i \"/src/.*\\.spec\\.ts$\" --force-exit",
+    "test": "run-s migration:test && NODE_ENV=test jest -i \"/src/.*\\.spec\\.ts$\" --verbose --runInBand --force-exit",
```

Comment:
Just curious, why the `--runInBand` now?

Reply:
Mainly to aid debugging purpose. it was quite hard to debug some slow tests without it. it proved useful for me. there's also some claims that it helps speed up tests but that seems negligible in our case.